### PR TITLE
Initial Wagtail setup

### LINKED ISSUE #62: Install Wagtail
Follow these instructions for integrating Wagtail into an existing project: http://docs.wagtail.io/en/v2.6.1/getting_started/integrating_into_django.html

### PR DESCRIPTION
**Pull request by beamalsky**
## Overview

Closes #62 

This PR installs Wagtail and sets up what we'll need for a persistent media directory using lessons learned in [LISC](https://github.com/datamade/lisc-cnda/pull/76) and now documented in [deploy-a-site](https://github.com/datamade/deploy-a-site/blob/master/File-uploads.md). It also creates a first sample data fixture of CMS content.  

## Testing Instructions

* Follow instructions in the [README](https://github.com/datamade/committee-oversight) to get the site running locally
* Run migrations: `python manage.py migrate`
* Create a superuser with a username and password of your choice: `python manage.py createsuperuser`
* Load initial CMS content: `python manage.py load_cms_content`
* Run the app locally: `python manage.py runserver`
* Navigate to the CMS login: http://localhost:8000/cms/
* Make sure you see a Wagtail login screen and that your new user login works!
* Make sure the CMS content loaded property. You should see: 
	- A single page with the title "Committee Oversight Database"
	- This cow under the Images tab on the left sidebar:

flowercow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,3 +66,24 @@ Perform the following steps from your terminal.
...
+image files—run:
+
+```bash
+python manage.py dumpdata --natural-foreign --indent 2 -e core -e legislative -e committeeoversightapp -e contenttypes -e auth.permission -e wagtailcore.groupcollectionpermission -e wagtailcore.grouppagepermission -e wagtailimages.rendition -e sessions > committeeoversightapp/fixtures/initial_cms_content.json
+```
+
+This should update the `initial_cms_content.json` file in your `committeeoversightapp/fixtures`
+directory.
+
+To update the images, find your `media` folder copy the contents of `original_images`
```
Comment (hancush) [inline README.md]:
```suggestion
To update the images, find your `media` folder, and copy the contents of `original_images`
```

Comment (hancush) [reply README.md]: want to include the command to do that here?

File: committeeoversightapp/management/commands/load_cms_content.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Based on the Wagtail demo:
+# https://github.com/wagtail/bakerydemo/blob/master/bakerydemo/base/management/commands/load_initial_data.py
+import os
+from distutils.dir_util import copy_tree
```
Comment (hancush) [inline committeeoversightapp/management/commands/load_cms_content.py]: woah cool!

**Comment by beamalsky** [comment]
@hancush you should be able to look at those config files now. I didn't rebase this correctly and the changes from #64 are showing up in the diff here as well—sorry!

**Pull request merged**